example/cavity/Solid/plotDisplacement.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Dec 13 09:09:07 2020
+
```

Comment:
Maybe rename this file to something like `plot-displacement.py`? Similar example: https://github.com/precice/tutorials/blob/master/perpendicular-flap/plot-displacement.sh